tools/copyAssets.ts

```diff
@@ -4,4 +4,5 @@ import * as shell from 'shelljs';
 shell.cp('-R', 'src/views', 'dist/');
 shell.cp('-R', 'src/public/assets', 'dist/');
 shell.cp('-R', 'src/public/css', 'dist/');
-shell.cp('-R', 'src/resources', 'dist/');
+shell.cp('-R', 'src/middleware/translations', 'dist/');
+shell.cp('-R', 'src/middleware/translations', 'dist/middleware/translations');
```

Comment:
Is this copying the translation files to both the root of the compiled app and the translation dir? Do we need it in both locations?

Reply:
Removed the copy to root... That was a mistake when I was debugging an issue with translations which was affecting a few things.